assets/dev/js/editor/editor-base.js

```diff
@@ -284,7 +284,7 @@ export default class EditorBase extends Marionette.Application {
 				return false;
 			}
 
-			if ( ! this.widgetsCache[ widgetType ].commonMerged ) {
+			if ( ! this.widgetsCache[ widgetType ].commonMerged && ! this.widgetsCache[ widgetType ].atomic_controls ) {
```

Comment:
maybe extract `this.widgetsCache[ widgetType ]` to a variable?

Reply:
I don't think it is necessary.